lib/spack/spack/build_environment.py

```diff
@@ -556,9 +557,9 @@ def _set_variables_for_single_module(pkg, module):
     if sys.platform == "win32":
         m.nmake = Executable("nmake")
     # Standard CMake arguments
-    m.std_cmake_args = spack.build_systems.cmake.CMakePackage._std_args(pkg)
-    m.std_meson_args = spack.build_systems.meson.MesonPackage._std_args(pkg)
-    m.std_pip_args = spack.build_systems.python.PythonPackage._std_args(pkg)
+    m.std_cmake_args = spack.build_systems.cmake.CMakeBuilder.std_args(pkg)
+    m.std_meson_args = spack.build_systems.meson.MesonBuilder.std_args(pkg)
+    m.std_pip_args = spack.build_systems.python.PythonPipBuilder.std_args(pkg)
```

Comment:
I'm hoping that this PR will make it easier for us to stop using these attributes someday, although it might not happen right away.

Reply:
If we are not concerned about breaking external repositories these can be removed (preferably in a following PR).